automobile/openstreetmap-importer.md

```diff
@@ -33,6 +33,9 @@ pip install lxml pyproj shapely --user
 
 ### Windows
 
+As a prerequisite, `Microsoft Visual C++ Compiler for Python 9.0` or higher is required to build the `pyproj` dependency.
```

Comment:
This is correct, but I'm wondering if it couldn't be confusing to put the version 9.0 in the name given that on the Microsoft page the software name is `Microsoft Visual C++ Compiler for Python 2.7 `.